components/InterventionList.svelte

```diff
@@ -75,9 +75,9 @@
   {#each $gjScheme.features as feature, i (feature.id)}
     <AccordionItem
       bind:open={feature.properties.editing}
-      on:click={startEditing(feature.id)}
-      on:mouseenter={sidebarHover(feature.id)}
-      on:mouseleave={sidebarHover(null)}
+      on:click={() => startEditing(feature.id)}
+      on:mouseenter={() => sidebarHover(feature.id)}
+      on:mouseleave={() => sidebarHover(null)}
```

Comment:
Because all these event handlers actually do pass in an event object, I was getting some error here before. We can use this syntax to ignore the event param